cli/cmd/metrics.go

```diff
@@ -122,18 +122,25 @@ func getPodsFor(ctx context.Context, clientset kubernetes.Interface, namespace s
 	// objects in Public API protobuf form for submission to the Public API
```

Comment:
Maybe remove/update the TODO as it talks about `BuildResource` which we are not using anymore?